sklearn/svm/_classes.py

```diff
@@ -1316,3 +1319,227 @@ def probA_(self):
     @property
     def probB_(self):
         return self._probB
+
+
+class SVDD(OutlierMixin, BaseLibSVM):
+    """Support Vector Data Description for Unsupervised Outlier Detection.
+
+    Estimate the support of a high-dimensional distribution by finding the
+    tightest soft hypersphere around a data set, which permits at most a
+    fraction ``nu`` (``0 < nu <= 1``) of the data as outliers.
+
```

Comment:
It would be great to insert one or two lines about the impact of the choice of the kernel on the shape of the decision function here (if possible).